test/integration/guest_user_test.rb

```diff
@@ -92,6 +93,12 @@ class GuestUserTest < ActionDispatch::IntegrationTest
     refute page.has_content?(non_tenant_item.title)
   end
 
+  test "will be redirected to home page if tenant does not exist" do
+    visit tenant_path(slug: "made-up-shop")
+
```

Comment:
No, we haven't talked about that. I was told that having all incorrect routes rerouted to root using a splat operator is sloppy. We can ask for guidance tomorrow!